pgpubsub/models.py

```diff
@@ -15,13 +15,20 @@
 class Notification(models.Model):
     channel = models.CharField(max_length=MAX_POSTGRES_CHANNEL_LENGTH)
     payload = JSONField()
+    # The field is made nullable to make sure the addiion of the new field is backward
+    # compatible. From the version this change is release the field is effectively non
+    # nullable as in it always gets a value.
+    # After some time the field should be made non nullable here.
+    created_at = models.DateTimeField(null=True)
 
     def __repr__(self):
         return (
             f'Notification('
-            f'  channel={self.channel},'
-            f'  payload={self.payload}'
-            f')')
+            f'channel={self.channel},'
```

Comment:
Not that the indentation was correct before this change, but it seems a little odd that `channel` is indented differently to the subsequent arguments

Reply:
It seems that it does not make sense to indent the first attribute because this does not indent the output as we do not not linebreaks. If we have it just adds a space after the parenthesis that is the result is:
```
Notification( channel=pgpubsub_a83de, payload={"app" : "tests", "model" : "Media", "old" : null, "new" : {"id":5,"name":"avatar.jpg","content_type":"image/png","size":15000}}, created_at=2023-05-06 03:59:26.563404)
```
versus

```
Notification(channel=pgpubsub_a83de, payload={"app" : "tests", "model" : "Media", "old" : null, "new" : {"id":5,"name":"avatar.jpg","content_type":"image/png","size":15000}}, created_at=2023-05-06 03:59:26.563404)
```
Do you prefer the first option?